File: FrameIncam.Client.FreeLancer/ng/domains/master/lov/masterLov.js
(function ($, ng, debugMode, undefined) {
    "use strict";

    // --------------------------------------------------------------------------------
    // Dependencies
    // --------------------------------------------------------------------------------
    var module = ng.module("app");

    // --------------------------------------------------------------------------------
    // Model
    // --------------------------------------------------------------------------------
    module.run([
        "services.utils",
        "services.models",
        function (
            p_utils,
            p_models
        ) {
            // --------------------------------------------------------------------------------
            // Namespace
            // --------------------------------------------------------------------------------
            var fincam = p_models.namespace("fincam");
            var lov = p_models.namespace("fincam.master.lov");

            // --------------------------------------------------------------------------------
            // Model
            // --------------------------------------------------------------------------------
            p_models.class({
                namespace: lov,
                name: "masterLov",
                base: {
                    namespace: fincam,
                    name: "model"
                },
                properties: {
                    id:  "int",
                    desc: "string",
                    type:  "string",
                    parentId: "int",
                    isactive: "int"
                },
                prototype: new function () {
                    var self = this;
                    return self;
                }
            });
        }
    ]);

})(jQuery, angular, document.children[0].hasAttribute("debug"));

File: FrameIncam.Domains/Models/Master/Lov/MasterLov.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace FrameIncam.Domains.Models.Master.Lov
{
    [Table("master_lov")]
    public class MasterLov : Model
    {
        [Column("desc")]
        public string Desc { get; set; }
        [Column("type")]
        public string Type { get; set; }
        [Column("parent_id")]
        public int? ParentId { get; set; }
        [Column("isactive")]
        public Int16 Isactive { get; set; }
    }
}
